### PR TITLE
Complete Task 9

### PR DESCRIPTION
**Pull request by Mahdi2105**

### CODE REVIEW

File: __tests__/endpoint.tests/endpoint.test.js [hunk 1]
```diff
@@ -376,3 +376,47 @@ describe("articles", () => {
     });
   });
 });
+
+describe("comments", () => {
+  describe("Delete comment by Id", () => {
+    test("DELETE:200 deletes a comment and returns the deleted comment", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(200)
```
Comment (pippaamy) [inline __tests__/endpoint.tests/endpoint.test.js:385]: a delete returns a 204 and you dont need to expect anything else except the status code

File: __tests__/endpoint.tests/endpoint.test.js [hunk 2]
```diff
@@ -376,3 +376,47 @@ describe("articles", () => {
...
+            votes: 16,
+          });
+        });
+    });
+
+    test("DELETE:404 returns error if comment id does not exist", () => {
+      return request(app)
+        .delete("/api/comments/928356")
+        .send({
+          inc_votes: -10,
```
Comment (pippaamy) [inline __tests__/endpoint.tests/endpoint.test.js:402]: we dont need to .send anything as we are doing a delete not a patch

File: __tests__/endpoint.tests/endpoint.test.js [hunk 3]
```diff
@@ -376,3 +376,47 @@ describe("articles", () => {
...
+        .then(({ body }) => {
+          expect(body.msg).toBe("Not found");
+        });
+    });
+
+    test("DELETE:400 returns error if comment id is not a number", () => {
+      return request(app)
+        .delete("/api/comments/not-a-number")
+        .send({
+          inc_votes: -10,
```
Comment (pippaamy) [inline __tests__/endpoint.tests/endpoint.test.js:414]: no need to send anything here either

File: controllers/comments.controllers.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const { deleteComment } = require("../models/comments.models");
+
+exports.removeComment = (req, res, next) => {
+  const id = req.params.id;
+  return deleteComment(id)
+    .then((comment) => {
+      res.status(200).send({ comment });
```
Comment (pippaamy) [inline controllers/comments.controllers.js:7]: we dont need to send the comment back here it can just be empty

File: models/comments.models.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      return db.query(
+        `
+      DELETE FROM comments 
+      WHERE comment_id = $1
+      RETURNING *;`,
+        [id]
+      );
+    })
+    .then(({ rows }) => {
+      console.log(rows[0]);
```
Comment (pippaamy) [inline models/comments.models.js:16]: delete console logs

**Review by pippaamy — COMMENTED** [review]
Great job - some changes needed in your tests. I've left some comments for you to have a look at :)

File: models/comments.models.js (hunk 5 above)
Comment (Mahdi2105) [reply models/comments.models.js:16]: Thanks for the feedback @pippaamy , did not even realise I was sending stuff on the deletes. I will make changes from both the PRs after I complete the current task I'm on :)

**Pull request merged**